modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -429,6 +432,9 @@ func (suite *TendermintTestSuite) TestUpdateState() {
 				suite.Require().True(path.EndpointA.GetClientState().GetLatestHeight().GT(clientMessage.GetHeight()))
 			},
 			func() {
+				bz := clientStore.Get(host.ClientStateKey())
+				updatedClientState := clienttypes.MustUnmarshalClientState(suite.chainA.App.AppCodec(), bz)
+
```

Comment:
should this go in malleate? I'm not sure these checks perform any actual checks now since `GetClientState()` performs the same code

Reply:
Right, yeah I get you! `GetClientState` is essentially just getting from the store and unmarshalling as well. So this is basically asserting true is true. 

Could you expand on what you're suggesting for moving to malleate? I'm not sure I follow, malleate will be called before `UpdateState` right, so the client state retrieved is not yet updated. Are you suggesting to retrieve the not yet updated state in malleate and then in expResult() assert that they are not equal?

Reply:
>  Are you suggesting to retrieve the not yet updated state in malleate and then in expResult() assert that they are not equal?

These test cases are asserting the client state didn't get updated (since we updated to a previous height). So yes, I am suggesting we construct the expected client state in malleate and then ensure the client state doesn't change after calling `UpdateState`

Reply:
Ah yes, sorry overlooked that when I wrote the comment. I've updated as you suggested to retrieve the "previous" states before update in malleate and check for equality in expResult! :))